src/qibocal/auto/transpile.py

```diff
@@ -97,19 +98,65 @@ def execute_transpiled_circuit(
         backend,
         transpiler,
     )[0]
-
     return transpiled_circ, backend.execute_circuit(
         transpiled_circ, initial_state=initial_state, nshots=nshots
     )
 
 
+def get_natives(platform):
+    """
+    Return the list of native gates defined in the `platform`.
+    This function assumes the native gates to be the same for each
+    qubit and pair.
+    """
+    pairs = list(platform.pairs.values())[0]
```

Comment:
Maybe singular is more meaningful...
```suggestion
    pair = list(platform.pairs.values())[0]
```